content/c-sharp/concepts/Conditionals/conditionals.md

```diff
@@ -0,0 +1,152 @@
+---
+Title: 'conditionals'
+Description: 'Compares inputs and gives an boolean output depending on it being True or False.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Dev'
+  - 'App Dev'
+  - 'Game Dev'
+Tags:
+  -
+CatalogContent:
+  - 'Learn C#'
+---
+
+## Explanation
+
+Below we'll follow the `if`, `else` and, `else if` statements one by one while expanding the example.
+
+## if statements
+
+An `if statement` compares two values and returns a value `if` true.
+
+## Example 1
+```cs
+
+var input1 = 10;
+var input2 = 10;
+var output;
+
+if (input1 === input2){
+    output = true; // sets the value of output to true.
+}
+```
+
+If the code above did not return true the code below will return false.
+
+```cs
+if(output === true){
+    Console.WriteLine('I returned true')
+}
+
+```
+>**note** `===` means equal. `!==` means not equal. 
+
+## Explanation of example 1
```

Comment:
These explanations don't require their own header. I suggest adding the explanation right underneath the code block. It's best to make the explanation simply and straightforwardly.